doc/api/v8.md

```diff
@@ -491,7 +491,7 @@ A subclass of [`Serializer`][] that serializes `TypedArray`
 (in particular [`Buffer`][]) and `DataView` objects as host objects, and only
 stores the part of their underlying `ArrayBuffer`s that they are referring to.
 
-### class: v8.DefaultDeserializer
+### class: `v8.DefaultDeserializer`
```

Comment:
what about using a capitalized `Class` to keep consistency?

Reply:
Good catch! Fixed! Thanks!